ai-library/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- crd.yaml
+- service-account.yaml
+- role.yaml
+- role-binding.yaml
+- operator.yaml
+namespace: opendatahub
+commonLabels:
+  opendatahub.io/component: "true"
+  component.opendatahub.io/name: ailibrary
+images:
+- name: quay.io/opendatahub/ai-library-operator
```

Comment:
Do we need to specify images here? Whats the purpose, because I see the same image name in the operator.yaml file

Reply:
If you change the image here, it will override what is in the other yaml files.  ie:  to update or kustomize it beyond the base yamls, you can just tweak this either here or in a overlay.

Reply:
https://github.com/kubernetes-sigs/kustomize/blob/master/docs/plugins/builtins.md#field-name-images

Reply:
@crobby so you have it here as just a placement for future changes? because today the image names match

Reply:
Yeah, I figured it's nice to have something there to tweak rather than needing to look up what needs to be added to tweak it.